testing/tests/DevExpress.data/odataStore.tests.js

```diff
@@ -506,6 +506,31 @@ QUnit.test("with expand", function(assert) {
         .always(done);
 });
 
+QUnit.test("expand should be undefined if it is not specified in the extraOptions", function(assert) {
+    assert.expect(1);
+
+    var done = assert.async();
+
+    ajaxMock.setup({
+        url: "odata2.org(42)",
+        callback: function(bag) {
+            this.responseText = { d: { expandClause: bag.data.$expand } };
```

Comment:
Is it possible to make here an assertion? `assert.equal(bag.data.$expand, undefined)`